hive-metastore/src/test/java/org/apache/iceberg/hive/HiveTableTest.java

```diff
@@ -214,29 +225,37 @@ public void testDropTable() throws IOException {
 
     List<ManifestFile> manifests = table.currentSnapshot().allManifests(table.io());
 
-    Assert.assertTrue(
-        "Drop (table and data) should return true and drop the table",
-        catalog.dropTable(TABLE_IDENTIFIER));
-    Assert.assertFalse("Table should not exist", catalog.tableExists(TABLE_IDENTIFIER));
-
-    Assert.assertFalse("Table data files should not exist", new File(location1).exists());
-    Assert.assertFalse("Table data files should not exist", new File(location2).exists());
-    Assert.assertFalse(
-        "Table manifest list files should not exist", new File(manifestListLocation).exists());
+    Assertions.assertThat(catalog.dropTable(TABLE_IDENTIFIER))
+        .as("Drop (table and data) should return true and drop the table")
+        .isTrue();
+    Assertions.assertThat(catalog.tableExists(TABLE_IDENTIFIER))
+        .as("Table should not exist")
+        .isFalse();
+
+    Assertions.assertThat(new File(location1).exists())
```

Comment:
for all these assertions against a File, those can be improved to `assertThat(...).exists()` because this is more expressive than just checking for a true/false

Reply:
```
org.opentest4j.AssertionFailedError: [Table data files should not exist] 
Expecting value to be true but was false
```
It is failing as the check made by Java File APIs and AssertJ APIs are different

Reply:
Yeah, in this context, we can use `doesNotExist() `as we are evaluating `isFalse()`